src/components/TableRow/TableRow.tsx

```diff
@@ -0,0 +1,56 @@
+import { Link } from 'react-router-dom';
+
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  person: Person,
+  slug: string | undefined,
+};
+
+export const TableRow: React.FC<Props> = ({ person, slug }) => {
+  return (
+    <>
+      <tr
+        data-cy="person"
+        className={classNames({
+          'has-background-warning': slug === person.slug,
+        })}
+      >
+        <td>
+          <Link
+            to={`../${person.slug}`}
+            className={classNames({
+              'has-text-link': person.sex === 'm',
+              'has-text-danger': person.sex === 'f',
+            })}
+          >
+            {person.name}
+          </Link>
```

Comment:
You can use here your PersonLink component

Reply:
check this comment your mate please